### ISSUE TITLE
not using gpu when training

### ISSUE THREAD

**Issue by toninhoautopecas** [Draichi/altcoin-bot-advisor#15]
I am trying to train my model but it's not using any of the gpus available, I am wondering if it's on purpose or an error on your code which I can't find.

OS: Ubuntu 16.04
GPU: 2x V100

![image](https://user-images.githubusercontent.com/57491435/68394265-4a112000-014c-11ea-9fd0-349e294f1287.png)

**Comment by Draichi** [comment]
RL algorithms are inherently inefficient when utilizing GPU. GPUs are best utilized when doing large batches of gradient descent. The issue is that most wall-clock time is spent doing inference within the simulation, which utilizes relatively little GPU. When enough data is collected, then the GPU utilization will go up for the actual gradient descent phase. You'll notice 20%~30% GPU usage at this time (if you have tensorflow-gpu installed and configured).

**Comment by toninhoautopecas** [comment]
Thanks for explanation, I'm new into reinforcement learning..
